PVTPackage/source/MultiphaseSystem/PhaseSplitModel/DeadOilFlash.cpp

```diff
@@ -34,9 +36,34 @@ DeadOilFlash::DeadOilFlash( std::vector< std::vector< double > > const & PVDO,
   m_gasPhaseModel( pvt::PHASE_TYPE::GAS, PVDG, gasSurfaceMassDensity, gasSurfaceMolecularWeight ),
   m_waterPhaseModel( PVTW, waterSurfaceMassDensity, waterSurfaceMolecularWeight )
 {
+}
 
+DeadOilFlash::DeadOilFlash( std::vector< std::vector< double > > const & PVDO,
+                            double oilSurfaceMassDensity,
+                            double oilSurfaceMolecularWeight,
+                            std::vector< std::vector< double > > const & PVDG,
+                            double gasSurfaceMassDensity,
+                            double gasSurfaceMolecularWeight )
+  :
+  m_oilPhaseModel( pvt::PHASE_TYPE::OIL, PVDO, oilSurfaceMassDensity, oilSurfaceMolecularWeight ),
+  m_gasPhaseModel( pvt::PHASE_TYPE::GAS, PVDG, gasSurfaceMassDensity, gasSurfaceMolecularWeight ),
+  m_waterPhaseModel()  
```

Comment:
So this `m_waterPhaseModel` is not needed...

Reply:
Yes currently `m_waterPhaseModel` is not used for a two-phase oil-gas simulation, but still has to be initialized somehow. That is why I added this call to the default constructor here.